onnx/test/test_external_data.py

```diff
@@ -206,8 +206,8 @@ def create_data_tensors(
     def create_test_model_proto(self) -> ModelProto:
         tensors = self.create_data_tensors(
             [
-                (self.attribute_value, "attribute_value"),
-                (self.initializer_value, "input_value"),
+                (self.attribute_value, "attribute_value"),  # type: ignore[list-item]
+                (self.initializer_value, "input_value"),  # type: ignore[list-item]
```

Comment:
Just curious: is it some new check brought by the newer ruff or black? (was wondering why the ignore isn't needed previously)

Reply:
mypy may be picking them up now that it knows the path I suppose